src/components/dropdown/dropdown.tsx

```diff
@@ -31,6 +31,7 @@ export interface DropdownProps {
   transparentBackground?: boolean;
   className?: string;
   toggleButtonClassName?: string;
+  selectListClassName?: string;
```

Comment:
Can we skip direct customizations by class names in favor of standardized behavior variants for list's width in accordance with Design System?
E.g. we have a `isListWidthLimited` prop for similar purpose.